lib/data_pull.rb

```diff
@@ -263,4 +266,46 @@ def run(args:, config:)
       )
     end
   end
+
+  class ProfileSummary
+    def run(args:, config:)
+      uuids = args
+
+      users = User.includes(:profiles).where(uuid: uuids).order(:uuid)
+
+      table = []
+      table << %w[uuid current_status activated_timestamp disabled_reason fraud_review_pending_timestamp fraud_rejection_timestamp]
+
+      users.each do |user|
+        if user.profiles.any?
+          user.profiles.sort_by(&:id).each do |profile|
+            profile_status = profile.active ? 'Active' : 'Inactive'
```

Comment:
we're using snake_case for column names, do people expect capitalized values here? Like my guess would to keep lowercase
```suggestion
            profile_status = profile.active ? 'active' : 'inactive'
```

Reply:
notes from our chat earlier today, I think we probably want to check for GPO pending and In Person Proofing Pending states, and this "status" column is a good spot for those results

Reply:
We will get "gpo_verification_pending" and "in_person_verification_pending" in disabled_reason column (profile.deactivation_reason)
![image](https://github.com/18F/identity-idp/assets/109746710/6d24d10a-73b2-4c44-874c-c40b03ca02bd)